_plugins/sitemap_generator.rb

```diff
@@ -1,312 +0,0 @@
-# Sitemap.xml Generator is a Jekyll plugin that generates a sitemap.xml file by
```

Comment:
From what I can tell, this plugin isn't currently wired up (there's no `sitemap.xml` generated for the published site). It was generating errors and warnings locally so just removing it.

Reply:
I can confirm that Google is not aware of any sitemaps for the blog:

https://search.google.com/u/1/search-console/sitemaps?resource_id=https%3A%2F%2Fartsy.github.io%2F

Which adds up because this is a 404:

https://artsy.github.io/robots.txt